include/arbor/simd/avx.hpp

```diff
@@ -8,8 +8,8 @@
 #include <cstdint>
 #include <immintrin.h>
 
-#include <simd/approx.hpp>
-#include <simd/implbase.hpp>
+#include <arbor/simd/approx.hpp>
+#include <arbor/simd/implbase.hpp>
```

Comment:
Why is SIMD moving to the public API? Of all the things to hide from users, that would intuitively be at the top of my list!

Reply:
This has to be exposed so that `modcc` users can compile their modules.

Reply:
(More than just simd has to be exposed for this, of course.)